gdrive/analytics_api.py

```diff
@@ -0,0 +1,164 @@
+"""
+Google Analytics Rest API
+"""
+
+from datetime import datetime
+import logging
+
+import fastapi
+from pydantic import BaseModel
+from fastapi import BackgroundTasks, responses
+import pandas as pd
+
+from gdrive import error, settings, analytics_client, sheets_client, drive_client
+
+log = logging.getLogger(__name__)
+router = fastapi.APIRouter()
+
+
+class AnalyticsRequest(BaseModel):
+    startDate: str = None
+    endDate: str = None
+
+
+@router.post("/analytics")
+async def run_analytics(background_tasks: BackgroundTasks):
+    background_tasks.add_task(run_analytics_task, datetime.today(), None)
```

Comment:
how long does this take? does it need to be async with respect to the http request?

Reply:
It takes a few seconds. Why would it be preferable to do this synchronously?

Reply:
@dzaslavskiy